### ISSUE TITLE
Hitting backspace key in data browser crashes dashboard

### ISSUE THREAD

**Issue by mtrezza** [parse-community/parse-dashboard#2453]
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-dashboard/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the latest versions of [Parse Server](https://github.com/parse-community/parse-server/releases) and [Parse Dashboard](https://github.com/parse-community/parse-dashboard/releases). 

### Issue Description

Hitting backspace key in data browser crashes dashboard

with error:

```
[Error] TypeError: undefined is not an object (evaluating 'this.props.value._previousSave')
value — dashboard.bundle.js:104940:115
```

at line:

```
if (this.props.value !== e.value && (this.renderCellContent(), null === (t = this.props.value._previousSave) || void 0 === t || null === (n = t.then((function() {
```

Bug introduced by https://github.com/parse-community/parse-dashboard/pull/2422.

### Steps to reproduce

1. Open data browser
2. Select deletable field
3. Hit backspace key

### Actual Outcome

Error thrown, window goes blank.

### Expected Outcome

Delete field value.

### Environment

Dashboard
- Parse Dashboard version: `5.2.0-alpha.11`

**Comment by mtrezza** [comment]
@patelmilanun Maybe you have time to take a quick look at this; the bug has been introduced by https://github.com/parse-community/parse-dashboard/pull/2422, and maybe it's quickest for you to look at this since you worked on the code recently. Thanks!

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.2.0-alpha.14](https://github.com/parse-community/parse-dashboard/releases/tag/5.2.0-alpha.14)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.2.0-beta.2](https://github.com/parse-community/parse-dashboard/releases/tag/5.2.0-beta.2)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.2.0](https://github.com/parse-community/parse-dashboard/releases/tag/5.2.0)
